main.js

```diff
@@ -0,0 +1,68 @@
+let storedBooks = [];
+const list = document.querySelector('.bookList');
+const form = document.querySelector('form');
+const title = document.querySelector('#bookTitle');
+const author = document.querySelector('#bookAuthor');
+
+function removebook(bookinfo, index) {
+  const bookData = document.getElementById(index);
+  storedBooks = storedBooks.filter((item) => item !== bookinfo);
+  localStorage.setItem('bookCollection', JSON.stringify(storedBooks));
+  list.removeChild(bookData);
+}
```

Comment:
- [ ] There is a bug in the code that is preventing you from achieving the desired functionality.  Allow me to explain with a video:

https://share.getcloudapp.com/9ZuDppz1

It seems that the localStorage is not updating well for deleting books unless you reload the page first.  After adding a book, you need to update the page and _then_ delete it otherwise it does not get deleted from localStorage.

Kindly please fix this bug in order to achieve the correct delete functionality.  You are very close! 💪